.github/workflows/ci.yml

```diff
@@ -110,32 +110,26 @@ jobs:
           sudo apt-get update
           sudo apt-get install --no-install-recommends -y tree libasound2-dev libglib2.0-dev libxcb-shape0-dev libxcb-xfixes0-dev \
             libcairo-dev libgtk2.0-dev libsoup2.4-dev libgtk-3-dev libwebkit2gtk-4.0-dev xorg-dev ninja-build libxcb-render0-dev
+      - name: Install protoc
+        uses: taiki-e/install-action@v2
+        with:
+          tool: protoc@3.21.12
       - name: Install run (headless) dependencies
         run: |
           sudo add-apt-repository ppa:oibaf/graphics-drivers -y
           sudo apt-get update
           sudo apt install -y libxcb-xfixes0-dev mesa-vulkan-drivers
 
-      - name: Cargo cache
-        uses: actions/cache@v3
+      - name: Rust cache
+        uses: Swatinem/rust-cache@v2
```

Comment:
Won't this use the same cache for this build as the other builds?

Reply:
With this setup only golden image runs share caches with each other. But for example test-linux job does not. Here I am experimenting if this cache implementation does a better job than what we had before, so I didn't change other jobs yet.

Reply:
Did we decide if this was better or not?

Reply:
Sorry, forgot to update this thread. Yeah I think it works quite well and it is less definitions than the previous one. It was recommended by the author of rust-analyzer and they use it too.